tasks/task02/task02.py

```diff
@@ -0,0 +1,59 @@
+import math
+print("Welcome to the triangle area calculation tool.")
+menu = '''Menu:
+1. Calculate triangle area by base and height
+2. Calculate triangle area by 2 sides and angle between them
+3. Exit
+'''
```

Comment:
it would be nice to keep the menu items numbers together with their explanations that should be shown to a user in menu. when you can loop over it to show available options.
this will make obvious why do we expect exactly those option numbers in the code. for this purpose you can use a dictionary.

Reply:
btw have you run the code?) if so you should have seen that there are exactly the numbers with their explanations) in this task also there were no prerequisite to use any data structure. so I appreciate your comment very much but I've just followed the task

Reply:
yes, I run it.
I mean it would be more understandable from the code perspective